src/lib/ensure-repo.zsh

```diff
@@ -16,6 +16,9 @@
     local update=${2:-false}
     # Verbose output.
     local verbose=${3:-false}
+    
+    # Ensure log file exists
+    [ ! -e $_ANTIGEN_LOG_PATH ] && touch $_ANTIGEN_LOG_PATH
```

Comment:
Rather than this it's probably better to use `&>>!` as per http://zsh.sourceforge.net/Doc/Release/Redirection.html in line [30](https://github.com/zsh-users/antigen/pull/358/files#diff-763b6d5d5863854eeeb837bd1c41e282R30).

Let me know if you're up to it, otherwise I'll do it.